cmd/statusd/main.go

```diff
@@ -42,6 +45,10 @@ var (
 	standalone = flag.Bool("standalone", true, "Don't actively connect to peers, wait for incoming connections")
 	bootnodes  = flag.String("bootnodes", "", "A list of bootnodes separated by comma")
 
+	// stats
+	statsEnabled = flag.Bool("stats", false, "Expose node stats via /debug/vars expvar endpoint or Prometheus (log by default)")
+	statsAddr    = flag.String("stats.addr", ":8080", "HTTP address with /debug/vars endpoint")
```

Comment:
I would be cool to provide the default in the description.
Also, should it be `0.0.0.0:8080` (as it now, accessible to anyone on the network) or `127.0.0.1:8080` (accessible only from localhost)?

Reply:
`127.0.0.1:8080` is safer but I kinda follow go-ethereum defaults. I shouldn't do that so I will change it.

Reply:
forgot to add, that this target, wnodetestcluster_wnode_3:8545, will work if the socket will be listening on 0.0.0.0:8545 or on IP that is assigned to eth0